rero_ils/modules/holdings/api.py

```diff
@@ -183,7 +183,8 @@ def delete(self, force=False, dbcommit=False, delindex=False):
         if can:
             if self.is_serial:
                 # Delete all attached items
-                for item in self.get_items:
+                for item_pid in Item.get_items_pid_by_holding_pid(self.pid):
+                    item = Item.get_record_by_pid(item_pid)
```

Comment:
I understand why you change this code.
But my feeling is that `get_items()` functions  should return all related items and not just standard or received items. Should be better to create a property `def items()` returning all items related to this holdings and a function `def get_items(filters)` that will return only items matching criteria.

So in your code only change `for item in self.get_items` to `for item in self.items`.

Reply:
Actually the command of the `get_items` function says: """Return standard items and received issues for a holding record.""". I will implement a function `items` as demanded.